_posts/2018-02-21-symfony-route-test.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Testing the new symfony router
+author: Georgos Landin Xanthopoulos
+date: '2018-02-02 15:06:47 +0200'
+header:
+  teaser: images/posts/routelarge.jpg
+  caption: "Photo credit: [**Flickr**](https://www.flickr.com/photos/rockys_photos/355216949/in/photolist-6enFxo-KGos4s-bziePy-91523N-TpGWrG-dvMSg3-6ZREpd-xozCa-3CVfrH-4r4Dz7-pkqvKY-5CL4FN-fHW3WW-8vvEYu-4KfNRw-6xwQuD-6xHB5q-8xo6sb-7dphxn-7KgWrr-342fEp-6xX2tf-6dBTjp-9nB8TA-8xk5pz-P61pjV-dNXAvC-5n8o7A-u7GL9y-6jzRhi-MGt7o1-7A7jLS-i4tC8C-4hDeBQ-3eX22G-dvGvj4-7A3gUD-5b6sLw-5yjxW2-8hxfmE-4cTxKp-Cq8sKM-tQg5e-7A7m1d-6ygJN3-avLA1p-9mQJiX-8ZtQtV-rpFoVZ-7A1DLZ)"
+categories:
+- Symfony
+---
+<b>How much faster is the new Symfony router in a real world application?</b>
+
+<b>We ran some tests to find out.</b>
+
+## An improved router
+
+A few days ago we received the fantastic news that the Symfony router had been improved significantly. Of course we wanted to see how
+the changes would affect our application, so we decided to test it against our own routes. 
+
+If you somehow missed the news you can read the official announcement here: 
+<https://symfony.com/blog/new-in-symfony-4-1-fastest-php-router>
+
+If you want to know how the new router works, read Nicolas Grekas [post](https://medium.com/@nicolas.grekas/making-symfonys-router-77-7x-faster-1-2-958e3754f0e1) where he dives more into detail about the new changes.  
```

Comment:
maybe also link to the 2nd part somewhere? :)
https://medium.com/p/making-symfony-router-lightning-fast-2-2-19281dcd245b

Reply:
Absolutely! Will add that :)